test/conftest.py

```diff
@@ -22,32 +22,35 @@ def log_enabled():
 def app_config() -> dict:
```

Comment:
rename to app_config_default?

Reply:
You can override it - and then you'd have to create an override called `app_config_default`.

Reply:
ok, got it. Then it is better to leave it like this.